gh-repo-export

```diff
@@ -231,10 +256,36 @@ download_migration() {
   fi
 }
 
+print_archive_url() {
+  local MIGRATION_ID="$1"
+  local MIGRATION_STATE="${MIGRATIONS[$MIGRATION_ID]}"
+
+  if [[ $GH_HOST != "" ]]; then
+    if [[ "$GH_HOST" == "github.com" ]]; then
+      BASE_URL="https://api.github.com"
+    else
+      BASE_URL="https://$GH_HOST/api/v3"
+    fi
+  else
+    BASE_URL="https://api.github.com"
+  fi
+
+  if [[ "$MIGRATION_STATE" == "exported" ]]; then
+    local ARCHIVE_URL=$(curl -s -I -H "Authorization: bearer $TOKEN" "$BASE_URL/orgs/$ORGANIZATION/migrations/$MIGRATION_ID/archive" | tr -d '\r' | sed -En 's/^location: (.*)/\1/p')
```

Comment:
@timrogers : I'm not as intimately familiar with using `-I` in this use case versus making the request without `-L` flag.  Could you elaborate on `curl` behavior without it?

Reply:
To be frank, I didn't even spot this option. It's definitely a better way to do it!

Reply:
I've just done some more digging into this, and the thing that is hard is both (a) getting the redirect URL and (b) silencing `curl`'s output so you don't get something ugly like this:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   494  100   494    0     0   1754      0 --:--:-- --:--:-- --:--:--  1776
```

The default behavior is actually pretty good - without `-L`, it prints the redirect URL - but if you try to silence the progress with `-s`, then you lose the progress, but you also lose the redirect URL.

However! I've just found a solution: `--no-progress-meter`. That seems to do the trick!